Artsy/Networking/API_Modules/ARUserManager.h

```diff
@@ -18,6 +18,12 @@ extern NSString *const ARUserSessionStartedNotification;
 - (User *)currentUser;
 - (void)storeUserData;
 
+@property (nonatomic, strong) NSString *appleEmail;
+@property (nonatomic, strong) NSString *appleDisplayName;
```

Comment:
I would say these should be readonly, since the getters are computed anyway 👍 

```suggestion
@property (nonatomic, readonly) NSString *appleEmail;
@property (nonatomic, readonly) NSString *appleDisplayName;
```